internal/scaffold/olm-catalog/csv_updaters.go

```diff
@@ -110,7 +123,14 @@ func (store *updaterStore) AddRole(yamlDoc []byte) error {
 	return nil
 }
 
-func (store *updaterStore) AddClusterRole(yamlDoc []byte) error {
+func (store *updaterStore) AddClusterRole(yamlDoc []byte) error { //nolint:golint
+
+	/*
+		* todo(camilamacedo86): remove //nolint:golint after merge of https://github.com/operator-framework/operator-sdk/pull/2250
+		Skip golint because it will be changed here: https://github.com/operator-framework/operator-sdk/pull/2250
+		receiver name store should be consistent with previous receiver name s for updaterStore (golint)
```

Comment:
This should be fixable by renaming `store` to `s`. Is there a reason we aren't doing that?

Reply:
the above PR was merged. So, i will be working on it and let you know.